src/controller/python/chip/utils/CommissioningBuildingBlocks.py

```diff
@@ -156,12 +161,14 @@ async def AddNOCForNewFabricFromExisting(commissionerDevCtrl, newFabricDevCtrl,
         newNodeId (int): Node ID to use for the target node on the new fabric.
 
     Return:
-        bool: True if successful, False otherwise.
+        tuple: (bool, nocResp): True if successful, False otherwise, along with nocResp, rcacResp value.
```

Comment:
this actually returns a tuple of (bool, Optional[nocResp], Optional[rcacResp])

Reply:
You're right. I added this in the comment line